main.py

```diff
@@ -1,7 +1,64 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    create_genres(("Western", "Action", "Dramma"))
+    create_actors({
+        "actor1": {
+            "first_name": "George",
+            "last_name": "Klooney"
+        },
+
+        "actor2": {
+            "first_name": "Kianu",
+            "last_name": "Reaves"
+        },
+
+        "actress1": {
+            "first_name": "Scarlett",
+            "last_name": "Keegan"
+        },
+
+        "actor3": {
+            "first_name": "Will",
+            "last_name": "Smith"
+        },
+
+        "actor4": {
+            "first_name": "Jaden",
+            "last_name": "Smith"
+        },
+
+        "actress2": {
+            "first_name": "Scarlett",
+            "last_name": "Johansson"
+        },
+    })
+
+    # update
+    Genre.objects.filter(name="Dramma").update(name="Drama")
```

Comment:
you could also define update/delete/retrieve as separate fuctions to better structure your code.
On the same note you can have one function for creating both genres and actors